MekHQ/src/mekhq/campaign/parts/Refit.java

```diff
@@ -1546,6 +1535,8 @@ private int getTimeMultiplier() {
         case CLASS_F:
             mult = 5;
             break;
+        case CLASS_A:
+        case CLASS_B:
         default:
             mult = 1;
         }
```

Comment:
Put a `break;` here for good measure.

Reply:
Good idea, and I fixed the formatting for it too.